src/Events/Event.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace EightPoints\Bundle\GuzzleBundle\Events;
+
+use Symfony\Component\EventDispatcher\Event as BaseEvent;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Contracts\EventDispatcher\Event as ContractsBaseEvent;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+if (is_subclass_of(EventDispatcher::class, EventDispatcherInterface::class)) {
```

Comment:
maybe we can comment why this was done? that would be nice, or?

Reply:
good idea :+1: added comment